audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
Yes, the game will search for ienssed and will not find it and won't play anything - sound.log gets a File not found error line. It's not stupid technically when you're working with your own stuff, but when you're working with the WW assets you can't do anything with it.

I understand it's not an elegant solution but __it's the best you can do__.